ucsb-cs156-m23/proj-happycows-m23-10am-2
# ka - makes unhealthy cows cheaper to sell

Decreases sell value of cows with health below 100% for players. New profit calculation is performed by multiplying the market sell value by average cow health. 

To test: 
1. go to https://proj-happycows-kirrarista.dokku-05.cs.ucsb.edu/
2. buy some cows
3. sell come cows
4. go to Admin->Manage Jobs-> Set Cow Health For A Specific Commons
5. change cow health
6. go to your commons
7. sell some cows
8. notice the change in profits per cow sold

Closes #15
